TKeazirian.HTTPServer/Controller.cs

```diff
@@ -2,27 +2,50 @@ namespace TKeazirian.HTTPServer
 {
     public class Controller
     {
-        private const string NewLine = "\r\n";
-
         public string EchoRequestBody(string request)
         {
-            var body = Parser.ParseBody(request);
+            string responseStatus = Constants.Status200;
+            string responseHeaders = Parser.ParseHeaders(request);
+            string responseBody = Parser.ParseRequestBody(request);
+
+            ResponseBuilder responseBuilder = new ResponseBuilder();
+
+            var response = responseBuilder.BuildResponse(responseStatus, responseHeaders, responseBody);
+
+            return response;
+        }
+
+        public string SimpleGetNoBody(string request)
+        {
+            string responseStatus = Constants.Status200;
+            string responseHeaders = Parser.ParseHeaders(request);
+            string responseBody = Parser.ParseRequestBody(request);
```

Comment:
What would happen if something went wrong here, say the parsing fails due to a malformed request? Would the server still return OK?